prow/apis/prowjobs/v1/types.go

```diff
@@ -243,9 +243,9 @@ type ProwJobStatus struct {
 	// ProwJob.
 	JenkinsBuildID string `json:"jenkins_build_id,omitempty"`
 
-	// PrevReportState stores the previous reported prowjob state
+	// PrevReportState stores the previous reported prowjob state per reporter
 	// So crier won't make duplicated report attempt
-	PrevReportState ProwJobState `json:"prev_report_state, omitempty"`
+	PrevReportState map[string]ProwJobState `json:"prev_report_state, omitempty"`
```

Comment:
(this is actually a breaking change by renaming a string field to map... we probably will want versioned prowjobs? I'll name it to something else for now...)

Reply:
Well that's what v1 is supposed to be for... probably this package should actually be v1alpha1 @stevekuznetsov @cjwagner ..